src/transforms/transform-branch.ts

```diff
@@ -5,6 +5,7 @@ import { WebhookPayloadCreate } from "@octokit/webhooks";
 import { generateCreatePullRequestUrl } from "./util/pull-request-link-generator";
 import { GitHubInstallationClient } from "../github/client/github-installation-client";
 import { JiraBranchData, JiraCommit } from "src/interfaces/jira";
+const MAX_COMMIT_MESSAGE_LENGTH = 1024;
```

Comment:
I don't like the idea of having multiple places for this.  I mean, I don't like the idea that we're duplicating the logic between sync and webhook either...

Maybe this constant can sit in the env.ts file?